1.0.3 (#19)
rooseveltframework/crossplatform-killport: CHANGELOG.md

## 1.0.3

- Fixed a bug that caused port detection to not properly handle port termination for both IPv4 and IPv6 addresses.
- Fixed a bug that prevented some processes from being killed.

> - Added -v, -version, --version flags to see version.

Reviewer: ```suggestion
- Added `-v`, `-version`, and `--version` flags to see version.
```

> - Added tests.

Reviewer: ```suggestion
- Added tests.
- Updated dependencies.
```

## 1.0.3

> - Added mocha tests along with c8 coverage.

Reviewer: ```suggestion
```

> - Enhanced the logic to properly handle port termination for both IPv4 and IPv6 addresses.

Reviewer: ```suggestion
- Fixed a bug that caused port detection to not properly handle port termination for both IPv4 and IPv6 addresses.
```

> - Added functionality that kills all processes associated with the specified open port. Effectively manages scenarios where both IPv4 and IPv6 ports are in use simultaneously.

Reviewer: ```suggestion
- Fixed a bug that prevented some processes from being killed.
```
